### ISSUE TITLE
dataset 

### ISSUE THREAD

**Issue by 1953678487** [open-mmlab/mmdetection#7574]
The number of pictures in my training set is 1440 , test set is 360and the number of samples per GPU is 2. Why is the number of samples behind the epoch printed by training (50/2160),(100/2160).....? Is it because of data enhancement? I use the official fasterrcnn.py for training and convert the data into VOC format.

**Comment by 1953678487** [comment]
in my understand,it should be (50/720) (100/720)........,i do not know 2016 mean

**Comment by jbwang1997** [comment]
Because the default VOC dataset uses the `RepeatDataset` to shorten the time of dataset preparation and checkpoint saving. The dataset has been repeated 3 times. So the number of iterations is 1440 / 2 * 3 = 2160.

**Comment by 1953678487** [comment]
thanks
